src/services/StructuralMetadataUtils.js

```diff
@@ -177,12 +156,28 @@ export default class StructuralMetadataUtils {
    * This mutates the state of the data structure
    */
   dndHelper = {
+    addSpanBefore: (clonedItems, wrapperSpanBefore) => {
```

Comment:
@Dananji Mind changing the parameter `clonedItems` to `allItems`.  The convention (not documented anywhere...just what I was originally thinking), is the `clonedItems` variable is usually created within a function, with an aim to not mutate the state of a passed in array of objects.  So if you make adjustments to the array anywhere, it's just a local mutation.

Maybe we keep functions accepting `allItems` as part of the function parameters, and if we need to clone it within the function, then `clonedItems` gets created.

Reply:
@adamjarling  For the `addSpanAfter` function, do we pass `allItems` for that?

Reply:
Yea, let's make that a consistent switch anywhere a function is being passed `clonedItems`.